modules/broadcast/src/backend/db.ts

```diff
@@ -124,38 +124,46 @@ export default class BroadcastDb {
       })
   }
 
-  async getUsersTimezone() {
+  async getUsersTimezone(): Promise<number[]> {
     const attrs = await this.knex('srv_channel_users').select('attributes')
-    const timezones = attrs.map(({ attributes: { timezone } }) => timezone)
+    const timezones = attrs.map(({ attributes: value }) =>
+      // SQLite returns a JSON string
+      typeof value === 'string' ? JSON.parse(value)?.timezone : value.timezone
```

Comment:
that's probably because we don't use our knex  extended `db.json.set`  to insert the value. Is it ?

Reply:
No we are using it when inserting/updating user's attributes: https://github.com/botpress/botpress/blob/master/src/bp/core/repositories/users.ts#L185

I guess that knex simply doesn't automatically serialize/deserialize JSON data with SQLite: https://github.com/knex/knex/issues/2575

I'll use `this.knex.json.get(attributes)` instead of directly calling json.parse.